src/idl_gen_general.cpp

```diff
@@ -1023,6 +1036,26 @@ void GenStruct(StructDef &struct_def, std::string *code_ptr) {
       code += lang_.getter_suffix;
       code += "}\n";
     }
+	// Generate ByKey accessor for vector of struct with key
+	if (field.value.type.base_type == BASE_TYPE_VECTOR &&
```

Comment:
If you make a seperate PR, and we merge that, then the change would automatically "dissapear" from the diff of this PR since it is already in there.